scripts/mkcloud

```diff
@@ -482,6 +491,13 @@ function setuphost()
         [[ $arch = s390x ]] && {
             echo 1 > /proc/sys/vm/allocate_pgste
         }
+
+        # enable UEFI boot
+        [[ -n $want_efi ]] && {
+            zypper --non-interactive in --no-recommends \
+                ovmf ovmf-tools qemu-ovmf-x86_64
+            [ "$?" == 0 -o "$?" == 4 ] || complain 10 "setuphost failed to install required packages"
```

Comment:
which should be fine, older host systems don't have ovmf available. I wouldn't make that a failure, but that can be debated.